test/symbolic/klee/nvim/keymap.c

```diff
@@ -236,13 +247,22 @@ static const struct key_name_entry {
   { K_PAGEUP,          "PageUp" },
   { K_PAGEDOWN,        "PageDown" },
   { K_KPAGEUP,         "kPageUp" },
+  { K_KPAGEUP,         "KP9" },
   { K_KPAGEDOWN,       "kPageDown" },
+  { K_KPAGEDOWN,       "KP3" },
+  { K_KORIGIN,         "kOrigin" },  // RFC: Correct name ?
```

Comment:
I don't think we need "RFC" here anymore.

Reply:
Ah, yes I forgot that one